### DISCUSSION TITLE
Welcome to bible-strong Discussions!

### DISCUSSION THREAD

**Discussion by smontlouis** [smontlouis/bible-strong#158] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JDArrOw3** [comment]
God bless you and the great work you are doing. Have an idea: What about Spanish translation? You can reach a lot more people. I can help u with it. arrowclaro@tutanota.com at your service as a volunteer.  May Jesus be with u

**Reply by smontlouis** [reply]
Hi, please create a new discussion so we can discuss about it :)

![image](https://github.com/user-attachments/assets/6c2c10ab-a2e2-4322-8b8a-fd412c0d3389)

Thank you for your help ! 

Regarding the spanish translation, the issue is not so much with the translations in-app, but the ressources (strong database, dictionary, nave, commentaries...) which can be challenging either to find or translate.

I think we can automate some with AI, but I'm not sure yet

**Comment by DerryDaniel-crypto** [comment]
Congratulations!
